pcmanfm/desktopentrydialog.ui

```diff
@@ -187,6 +209,13 @@
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="catEdit">
+       <property name="whatsThis">
+        <string>Semicolon separated categories in which the entry should be shown if it is inside the DE's main menu.
```

Comment:
inside the DE's main menu (usually saved in ~/.local/share/applications).  ?

Reply:
Yes. Why did you ask?

Reply:
I asked if we should also tell _how_ to add it to the menu, adding the path to the location in the text.

Reply:
Oh, that's really beyond this small utility. And if someone adds "Categories", they probably know what they'll do with it.